skins/shared/styles/buttons/buttons-wikia.scss

```diff
@@ -8,11 +8,11 @@ These buttons are used everywhere on our wikis.
 a.wikia-button,
 .wikia-single-button a,
 .wikia-menu-button,
-input[type="submit"],
-input[type="reset"],
-input[type="button"],
-.button,
-button {
+input[type="submit"]:not(.wds-button),
```

Comment:
Is this change related to the new special page?

Reply:
Yeah, otherwise 2010 era button styles will conflict with Design System buttons

Reply:
Alright, I'm doing that change to my 2018 A CSS sitewide pack too to avoid conflicts.

Reply:
this isn't really relevant to this conversation

Reply:
I know but thanks for fixing them so as not to conflict them